### PR TITLE
Journal table

### LINKED ISSUE #1: Synchronize features of mqtt-pg-logger and pgqueuer

### PR DESCRIPTION
**Pull request by agritheory**
@MyuddinKhatri Thanks for tracking down the issue, and finding some other cruft as well. The Frappe side of this integration will use [pgquerer](https://pgqueuer.readthedocs.io/en/latest/index.html) in the Frappe environment, which expects events to be in a table called `pgqueuer`. The table `journal is from the original design of this app, but the table name is flexible.

### CODE REVIEW

File: sql/table.sql [hunk 1]
```diff
@@ -51,6 +55,15 @@ CREATE TABLE pgqueuer_statistics (
 	entrypoint TEXT NOT NULL             -- The entrypoint function that processed the job.
 );
 
+CREATE TABLE journal (
```
Comment (agritheory) [inline sql/table.sql]: I don't think we need this block, all of these field already exist in the `pgqueuer` table above. I don't know if retain should be a Boolean or not.

File: src/message_store.py [hunk 2]
```diff
@@ -58,13 +58,13 @@ def store(self, messages):
         if not messages:
             return
 
-        copy_statement = sql.SQL("COPY {} (message_id, topic, text, qos, retain, time) FROM STDIN") \
+        copy_statement = sql.SQL("COPY {} (topic, text, qos, retain, time) FROM STDIN") \
```
Comment (agritheory) [inline src/message_store.py]: I think we need to add the other values that `pgqueuer` expects here. I am also confused as to why it's copying form STDIN, that seems like a bad design, but I just don't know. @vjFaLk?

File: src/message_store.py [hunk 3]
```diff
@@ -58,13 +58,13 @@ def store(self, messages):
...
             .format(sql.Identifier(self._table_name))
 
         with self._connection.cursor() as cursor:
             with cursor.copy(copy_statement) as copy:
                 for m in messages:
-                    data = (m.message_id, m.topic, m.text, m.qos, m.retain, m.time)
+                    data = (m.topic, m.text, m.qos, m.retain, m.time)
```
Comment (agritheory) [inline src/message_store.py]: Same as above RE making sure we're inserting all the values.

**Review by agritheory — COMMENTED** [review]
I think there's another line that needs to change to make this work, changing the default table name.
https://github.com/agritheory/mqtt-pg-logger/blob/535fc9eb5157b9c7ee4be7989476cacfa4c09c36/src/database.py#L62

**Comment by agritheory** [comment]
This PR is based on changes in progress in #5 and should be rebased after that is merged.

File: sql/table.sql (hunk 1 above)
Comment (MyuddinKhatri) [reply sql/table.sql]: Removed the journal block, but I am not fully sure but I think retain should be a Boolean.

File: src/message_store.py (hunk 2 above)
Comment (MyuddinKhatri) [reply src/message_store.py]: Updated the code add values according to `pgqueuer`

**Comment by agritheory** [comment]
@MyuddinKhatri Thanks for picking this up, it looks like @alchez has incorporated these changes into his work in #14.

**Pull request closed without merging**